frontend/src/pages/PassingTest/PassingTestContainer.tsx

```diff
@@ -116,17 +134,21 @@ const PassingTestContainer: React.FC = () => {
   const progressBarData = isFailed
     ? convertTestStatusToProgress(TEST_STATUS.failed, percentageValue)
     : convertTestStatusToProgress(TEST_STATUS.successful, percentageValue);
-  const [showDialogOnSwitchingRoute, setShowDialogOnSwitchingRoute] = useState<boolean>(false);
+  // const [showDialogOnSwitchingRoute, setShowDialogOnSwitchingRoute] = useState<boolean>(false);
 
-  const [showPrompt, confirmNavigation, cancelNavigation] = useCallbackPrompt(
-    showDialogOnSwitchingRoute,
-  );
+  // const [showPrompt, confirmNavigation, cancelNavigation] = useCallbackPrompt(
+  //   showDialogOnSwitchingRoute,
+  // );
```

Comment:
comments

Reply:
problems with pr, its should be at another branch